x/participationrewards/keeper/rewards_lockup_test.go

```diff
@@ -0,0 +1,77 @@
+package keeper_test
+
+import (
+	"cosmossdk.io/math"
+	"github.com/cosmos/cosmos-sdk/types"
+	"github.com/ingenuity-build/quicksilver/app"
+)
+
+func (s *KeeperTestSuite) TestAllocateLockupRewards() {
```

Comment:
should be `suite` rather than `s` to match the rest of the usages of `*KeeperSuiteTest`

Reply:
done